sbt-plugin/src/main/scala/scala/scalajs/sbtplugin/ScalaJSPluginInternal.scala

```diff
@@ -575,7 +576,8 @@ object ScalaJSPluginInternal {
 
       loadedJSEnv <<= Def.taskDyn {
         val log = streams.value.log
-        val libs = resolvedJSDependencies.value.data
+        val libs = resolvedJSDependencies.value.data :+
+            ResolvedJSDependency.minimal(javaSysProps2JSLib(javaOptions.value))
```

Comment:
I think assembling this file should be a separate task. The current implementation harms composability.